### DISCUSSION TITLE
Deadlines being added to README.md

### DISCUSSION THREAD

**Discussion by boyland** [community/community#52527] [GitHub Education] [answered]
Dear GitHub education/classroom

Recently GitHub Classroom has started adding a README.md with a deadline when the repo is cloned for a classroom invitation.  How do I turn off this behavior?  It messes up my stats of student commits.

**Comment by samus-aran** [comment, marked as answer]
Hi @boyland 👋 

Thanks for your feedback! The Classroom team has confirmed that the Classroom commit calculations that are showing on the interface excludes commits created by the classroom bot this including the additional bot created deadline commit. Are you monitoring the raw commits as a way to determine student stats? At this moment in time we don't provide the option to opt out of having the deadline info added to the readme whilst using the deadline feature.

Ask me anything to clarify 👍

**Reply by boyland** [reply]
Thanks for letting me know there is no way to avoid the commit except by not letting github classroom know the deadline.  The workaround that I am currently considering is to have my commit calcuation (which doesn't just count commits but compares sizes) ignore the first commit

**Reply by samus-aran** [reply]
Ah, so just for our team to understand you calculate your commits separately from how the existing Classroom interface presents them. I'm curious to ask what your calculation shows and how does it align with your teaching so we can gain an understanding and a need in the current product 🙂

**Reply by boyland** [reply]
We download all the repos for an assignment and run a script that computes the fewest number of commits that are bigger (in sum) than all the rest put together.  If this number is less than 3, then the student loses points for not doing sufficient fine-grained commits.  (Of course this can be gamed -- but the idea is to encourage people to commit after each edit session.)  It also checks that the student has more than a single commit.  The little commit that github classroom does won't influence the first computation much if there are lots of commits but it definitely influences the last check.
